src/AddTodoForm.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+
+const AddTodoForm = () => {
+    return(
+    <div>
+        <form>
+            <label htmlFor='todoTitle'> TITLE </label> <br /> <br />
+            <input type='text' id='todoTitle' value={"Time to add new tasks!"} />  <br /> <br />
```

Comment:
You shouldn't assign a string to that value attribute, if you try to change it later by writing in that input, you wouldn't be able to do it.